# Welcome to the TrynkaLab GitHub üëã  

We‚Äôre the **Trynka Group** at the **Wellcome Sanger Institute**, exploring how genetic variation shapes immune cell function and contributes to autoimmune diseases. By combining **genomics, computational biology, and experimental immunology**, we aim to decode how our DNA influences health and disease‚Äîand ultimately, to pave the way for better treatments.  

Here you‚Äôll find our open-source repositories, tools, and pipelines developed to advance the study of **immune cell regulation**, **genetic association**, and **multi-omics integration**.  
We love collaboration, reproducibility, and turning complex data into biological insight‚Äîso feel free to explore, reuse, or contribute! üß¨‚ú®  

---

## üì∞ Publication repositories  

- [Perez_Alcantara_pooled_iMGL](https://github.com/TrynkaLab/Perez_Alcantara_pooled_iMGL)  
- [VIDRA](https://github.com/TrynkaLab/VIDRA)  
- [Washer_et_al_microglia_media](https://github.com/TrynkaLab/washer_et_al_microglia_media)  
- [Omenn_paper](https://github.com/TrynkaLab/Omenn_paper)  
- [Treg_Multiomics](https://github.com/TrynkaLab/Treg_Multiomics)  
- [T-cell-costimulation](https://github.com/TrynkaLab/T-cell-costimulation)  
- [CellDivider](https://github.com/TrynkaLab/CellDivider)
---

## üî¨High content imagaging with TGlow

- **tglow-r** ‚Äî R package for HCI feature analysis: [tglow-r](https://github.com/TrynkaLab/tglow-r)  
- **tglow-pipeline** - A Nextflow pipeline for HCI processing: [tglow-pipeline](https://github.com/TrynkaLab/tglow-pipeline)
- **tglow-core** - Python package supporting the tglow-pipeline [tglow-core](https://github.com/TrynkaLab/tglow-core)
- **tglow-dino4cells** - Adaptation of DINO4Cells which produces multi-channel embeddings [tglow-dino4cells](https://github.com/TrynkaLab/tglow-dino4cells)
  
---

## üìä R packages  

- **poodleR** ‚Äî Donor unmixing using least squares: [poodleR](https://github.com/TrynkaLab/poodleR)  
- **ProliferationAnalysis** ‚Äî FACS proliferation analysis of complex mixtures in R: [ProliferationAnalysis](https://github.com/TrynkaLab/ProliferationAnalysis)
  
---

## ‚öôÔ∏è Nextflow pipelines  
- **sc-blipper** ‚Äî Single-cell post-processing running cNMF and gene set enrichments: [sc-blipper](https://github.com/TrynkaLab/sc-blipper)
  
---
## üß† Other software and tools  
- **CHEERS** ‚Äî Genetic enrichment of chromatin peaks: [CHEERS](https://github.com/TrynkaLab/CHEERS)  
